### DISCUSSION TITLE
Execution failed for task ':euphony:externalNativeBuildDebug'.

### DISCUSSION THREAD

**Discussion by phj0407** [euphony-io/euphony#142] [Q&A]
안녕하세요. 빌드 실패 관련하여 질문 드립니다! 

https://github.com/euphony-io/euphony/discussions/45
위 글을 참고하여 Build를 진행했더니 아래와 같이 실패가 떴습니다.

![image](https://user-images.githubusercontent.com/89901283/178423313-169c267b-5330-4403-a5e2-e6edc1417a96.png)
 
![image](https://user-images.githubusercontent.com/89901283/178423417-f3200409-469a-4d36-bacd-1828a0ca79f8.png)

나와있는 해결책을 따라 
1. NDK, CMake, platform-tools 설치 확인
 2. adb 경로 및 명령어 동작 확인
![image](https://user-images.githubusercontent.com/89901283/178423868-7e824342-9c83-44b4-b307-9580fdb48959.png)

 3. euphony/.cxx 삭제 시도 후 다시 build 했더니 

![image](https://user-images.githubusercontent.com/89901283/178422997-8b473261-ad66-480d-966e-c22d1ea40da9.png)
![image](https://user-images.githubusercontent.com/89901283/178425435-5f929032-2cb2-4855-9cca-3018d21dddd5.png)

이번에는 다른 실패 메시지가 나왔습니다😢 이유와 해결 방법을 알 수 있을까요?

**Comment by win-luck** [comment]
have you ever rebuilded your project? close all project in Android studio and reboot you computer, and build your project only.

**Comment by yeonns** [comment]
@phj0407 현재 발생한 에러로그에 대해 스샷을 첨부해주셨는데, 에러로그 전체를 텍스트로 붙여넣어주시고, 실행중인 환경에 대한 정보도 함께 기입해주시면 이슈 해결에 많은 도움이 될 거 같아요~ (ex. 안드로이드 스튜디오 버전, 연결된 에뮬레이터/기기 스펙 등등)

**Comment by designe** [comment]
음.. 
```java
java.io.IOException: 여기 문자 깨져 보이는게 심상치 않은 이슈네요.
```
gradle이 혜진님 폴더 아래에 있는데 혹시 혜진님 이름을 한글이 아닌 영어로 해도 동일 이슈가 발생하는지 궁금하네요.

**Reply by phj0407** [reply]
요 문제가 제일 유력한 것 같습니다! 파일 경로에 한글 없는 다른 기기로 돌렸더니 문제없이 돌아가네요ㅎㅎ... 수정 후 다시 시도해보겠습니다!!

**Comment by phj0407** [comment]
한글 경로 영어로 바꾸고 externalNativeBuildDebug파일에서 cmake 버전 맞춰주니 해결됐습니다ㅎㅎ!! 

`
